include/sddf/block/blk_shared_queue.h

```diff
@@ -0,0 +1,342 @@
+/*
+ * Copyright 2023, UNSW
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <stdint.h>
+#include <stddef.h>
+#include <stdbool.h>
+#include "fence.h"
+
+/* Number of buffers the request queue is configured to have. */
+#define BLK_REQ_QUEUE_SIZE 1024
+/* Number of buffers the response queue is configured to have. */
+#define BLK_RESP_QUEUE_SIZE 1024
+
+typedef struct blk_storage_info {
+    char serial_number[64];
```

Comment:
where does `64` come from?

Reply:
most serial numbers should fit within 64 characters, will leave a comment